### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by IhabBendidi**
Small update in main readme

### CODE REVIEW

**Comment by IhabBendidi** [comment]
You can delete branch afterward

File: README.md [hunk 1]
```diff
@@ -1,255 +1,15 @@
...
-# Name constraints
-
-Category names should only contain lower case characters `[a-z]`, dashes (`-`) and underscores (`_`).
-Whitespace characters, accented characters and punctuation signs (except for dash) are not allowed.
-
-Note that a category name shouldn’t be an empty string, shouldn’t contain two or more underscore characters or dashes in a row, and cannot start or end with an underscore character or a dash.
+# API Usage Guide
 
-For people names, we recommend using `lastname_firstname_middlename` format, though it is not a constraint.
+For the server API usage, you can find its documentation [here] (https://github.com/Zenika/zihub/tree/master/api)
```
Comment (egor-jerome) [inline README.md]: rather a link to the .md file

Comment (egor-jerome) [reply README.md]: README.md

Comment (IhabBendidi) [reply README.md]: okey on it

**Pull request merged**